grommet/grommet
# Fix RadioButton extend

#### What does this PR do?
Fixes RadioButton so that when `radioButton.extend` is used in the theme it is not applied to two DOM elements

#### Where should the reviewer start?

#### What testing has been done on this PR?
Testing with yarn storybook and yarn test

#### How should this be manually tested?
Add `RadioButton.extend: 'border: 1px solid red;'` to a theme and see that it is no longer being applied twice

#### Any background context you want to provide?

#### What are the relevant issues?
Closes #4172 

#### Screenshots (if appropriate)
Before:
<img width="178" alt="Screen Shot 2020-07-29 at 1 35 14 PM" src="https://user-images.githubusercontent.com/54560994/88844844-871cb700-d1a0-11ea-9aaf-63596bf9ae78.png">
After:
<img width="205" alt="Screen Shot 2020-07-29 at 1 37 09 PM" src="https://user-images.githubusercontent.com/54560994/88844907-9ef43b00-d1a0-11ea-8305-6426d00e0ac4.png">
After with container extend: 'border: 1px solid blue;'
<img width="250" alt="Screen Shot 2020-08-07 at 10 47 26 AM" src="https://user-images.githubusercontent.com/54560994/89668688-6f79b880-d89b-11ea-8845-73d05ba12ab8.png">

#### Do the grommet docs need to be updated?
Yes, updated in this PR

#### Should this PR be mentioned in the release notes?
Yes
Note: Previously `radioButton.extend` applied to the RadioButton and the container around the RadioButton. Now `radioButton.extend` only applies to the styling of the RadioButton and `radioButton.container.extend` applies to only the container.
To maintain any previous styling done with `radioButton.extend` add the styling to `radioButton.container.extend` as well.

#### Is this change backwards compatible or is it a breaking change?
Backwards compatible
